File: Examen1/2026_1/Resolucion/ExamenParcialCSharp/GestionAlumnosModel/Alumno/Alumno.cs
namespace GestionAlumnosModel.Alumno
{
    public class Alumno
    {

        private int id;
        private String codigo;
        private String nombre;
        private String apellidos;
        private String correo;
        private char estado;

        public int Id { get => id; set => id = value; }
        public string Codigo { get => codigo; set => codigo = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Apellidos { get => apellidos; set => apellidos = value; }
        public string Correo { get => correo; set => correo = value; }
        public char Estado { get => estado; set => estado = value; }
    }
}

File: Examen1/2026_1/Resolucion/TestSoft/testsoft-model/src/main/java/pe/edu/pucp/testsoft/model/Alumno.java
package pe.edu.pucp.testsoft.model;

public class Alumno {
    private int id;
    private String codigo;
    private String nombre;
    private String apellidos;
    private String correo;
    private char estado;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public char getEstado() {
        return estado;
    }

    public void setEstado(char estado) {
        this.estado = estado;
    }
}
